src/osd/strconv.h

```diff
@@ -25,22 +25,30 @@
 
 #include <windows.h>
 
-// the result of these functions has to be released with osd_free()
-
+std::string astring_from_utf8(const std::string &s);
 std::string astring_from_utf8(const char *s);
+std::string &astring_from_utf8(std::string &dst, std::string &s);
 std::string &astring_from_utf8(std::string &dst, const char *s);
+std::string utf8_from_astring(const std::string &s);
 std::string utf8_from_astring(const CHAR *s);
+std::string &utf8_from_astring(std::string &dst, const std::string &s);
 std::string &utf8_from_astring(std::string &dst, const CHAR *s);
 
+std::wstring wstring_from_utf8(const std::string &s);
 std::wstring wstring_from_utf8(const char *s);
+std::wstring &wstring_from_utf8(std::wstring &dst, std::string &s);
 std::wstring &wstring_from_utf8(std::wstring &dst, const char *s);
+std::string utf8_from_wstring(const std::wstring &s);
 std::string utf8_from_wstring(const WCHAR *s);
+std::string &utf8_from_wstring(std::string &dst, const std::wstring &s);
 std::string &utf8_from_wstring(std::string &dst, const WCHAR *s);
 
 #ifdef UNICODE
+typedef std::wstring tstring;
```

Comment:
Ok, I can remove it.  But this will necessitate reverting a bunch of 'tstring' declarations to 'auto', either that or stop using TCHAR (as per above)